src/db.rs

```diff
@@ -1270,6 +1270,97 @@ impl DB {
             Ok(())
         }
     }
+
+    /// Returns a list of all table files with their level, start key
+    /// and end key
+    pub fn livefiles(&self) -> Result<Vec<LiveFile>, Error> {
```

Comment:
Personally I would prefer to name this function `live_files`.

Reply:
I addressed your comment, PTAL :+1: